conf/application.conf

```diff
@@ -36,12 +52,26 @@ play.http.router = prod.Routes
 # The below controllers are the default exceptions to this rule.
 
 controllers {
-  uk.gov.hmrc.pillar2externalteststub.controllers.MicroserviceHelloWorldController = {
+  # 300 is the default, you may need to change this according to your needs
+  confidenceLevel = 300
+
+  uk.gov.hmrc.pillar2externalteststub.controllers.SubscriptionController = {
     needsAuth = false
     needsLogging = false
     needsAuditing = false
   }
 }
+# Metrics plugin settings - graphite reporting is configured on a per env basis
+metrics {
+  enabled = true
+}
+
+# Microservice specific config
+
+auditing {
+  enabled = true
```

Comment:
please turn this off locally

Reply:
Okay, it is enable in our existing stub though